lib/dynamic_paperclip/attachment_style_generator.rb

```diff
@@ -1,5 +1,5 @@
 require 'rack/request'
-require 'action_controller/metal/data_streaming'
+require 'action_dispatch/http/response.rb'
```

Comment:
It looks like both files exist in both versions, so we could require both, or we could check the Actionpack version and require the one we need.